plugins/command-line/prism-command-line.js

```diff
@@ -79,6 +62,21 @@
 		}
 
 		var codeLines = env.code.split('\n');
+
+		/** @type {int[]} */
+		var continuationLineIdxs = commandLine.continuationLineIdxs = [];
```

Comment:
Please don't abbreviate "indexes".

Also, this array is essentially used as a set of indexes. So why not use a [`Set`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set)?

Reply:
Will rename.
I don't do very much js so overlooked `Set` as a thing, also partly thought for the small number of lines involved it would not be much of a hit to see if it was in an array.  Happy to fix.